src/css/style.css

```diff
@@ -471,6 +470,24 @@ button {
     content: "\e60c";
 }
 
+.gpii-fd-dummy:before {
+    content: "\e60d";
+}
+
+.gpii-fd-navIcon-outer {
+    width: 30rem; /* Space for 5 icons */
+    overflow: hidden;
+    padding-top: 1.5rem; /* gap between main body and icons*/
+}
+
+.gpii-fd-navIcon-inner {
+    white-space: nowrap;
+    position: relative;
+    left: 0rem;
+    margin-left: 0;
+    width: 36rem; /* Space for 6 icons */
```

Comment:
Why does the inner container needs space for 6 icons while the outer is already restrained for 5 icons?